pagopa/io-app
# feat: [IOCOM-690] Move third party data request into its saga for SEND messages

⚠️ This PR depends on #5292  ⚠️

## Short description
This PR moves the request to get a SEND message third party data into its own saga.

## List of changes proposed in this pull request
- SEND message screen does not request third party data retrieval anymore
- Such request has been moved into `messageData` saga

## How to test
Using the io-dev-api-server, check that both SEND and remoted content messages retrieve their data properly
